entity-view/impl/src/main/java9/com/blazebit/persistence/view/impl/proxy/UnsafeHelper.java

```diff
@@ -1,38 +0,0 @@
-/*
- * Copyright 2014 - 2020 Blazebit.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package com.blazebit.persistence.view.impl.proxy;
-
-import java.lang.invoke.MethodHandles;
-
-/**
- *
- * @author Christian Beikov
- * @since 1.2.0
- */
-public final class UnsafeHelper {
```

Comment:
I'm pretty sure we still need the unsafe helper for Java 9? Included by accident?

Reply:
Yes this was included by accident.
I just pushed a new version of the commit.